apps/els_lsp/src/els_indexing.erl

```diff
@@ -112,6 +113,17 @@ index_references(#{uri := Uri} = Document, 'deep', false) ->
 index_references(_Document, 'shallow', _) ->
   ok.
 
+-spec maybe_start() -> true | false.
+maybe_start() ->
```

Comment:
It looks like the return value of this function is never used, so it could simply return `ok`.

Reply:
For my own education - is it preferred in the LS code base to return descriptive values only if they are used?

Reply:
I guess it's a matter of personal preference and it depends on the specific case. If we foresee a possible usage of the return value it makes sense to expose it, otherwise `ok` is quite common for a function whose purpose is mainly a side effect.